msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -32,21 +30,36 @@ You can use deep links in Teams in the following ways:
+* **Deep link to a workflow**: Use a deep link to create a new chat, open a scheduling dialog, or navigate to an audio-video call. App users can benefit from an improved app experience by utilizing simplified or automated user tasks. These tasks include initiating a new chat or scheduling a meeting that can be made easier by prepopulating the deep links with necessary parameters.
 
     For more information, see [deep link to a workflow](~/concepts/build-and-test/deep-link-workflow.md).
 
     :::image type="content" source="~/assets/images/deeplink-schedule.gif" alt-text="Graphical representation shows the user experience of deep links to launch a meeting scheduling dialog.":::
 
-You can use the Microsoft Teams JavaScript client library (TeamsJS) to navigate within your tab. For scenarios such as navigating to content and information or launching a chat dialog. The TeamsJS typed APIs provide an improved experience and are recommended for Teams apps that might run in Outlook or Microsoft 365 apps. They also provide a way to check that the capability used is supported by the host.
-
-<!--- TBD: Edit this article.
-* Admonitions/alerts seem to be overused. 
-* An important alert at the end of this table does not make sense. Also, it has a code snippet inside it.
-* List items in the table are not formatted well in output.
-* Some headings use -ing verbs.
-* Example values and some URLs should be in backticks and not emphasized.
-* Codeblock are missing language.
-* Check for markdownlint errors.
-* Table with just a row isn't really needed. Provide the content without tabulating it.
---->
+You can use the Microsoft Teams JavaScript client library (TeamsJS) to navigate within your tab. For scenarios such as navigating to content and information or launching a chat dialog, the TeamsJS typed APIs provide an improved experience. We recommend using TeamsJs typed APIs for Teams apps that might run in Outlook or Microsoft 365 apps. They also provide a way to check if the host supports the capability used.
+
+## Protocol handlers in deep links
+
+Teams deep links support two types of protocol handlers:
+
+1. **HTTPS**: The `https://` protocol handler is the default handler in most URLs. Using this protocol handler in a Teams deep link opens it in a browser window that gives you three options:
+    1. Open the deep link in the Teams desktop client if it's installed
+    2. Download the Teams desktop client if it's not installed
+    3. Open the deep link in the Teams web client
+
+    :::image type="content" source="../../assets/images/deep-link-open.png" alt-text="Screenshot shows a deep link opened in a browser.":::
+
+    > [!NOTE]
+    > A deep link must start with `https://teams.microsoft.com/l/` to ensure that it's handled correctly by Teams. Classic Teams URLs copied from the browser's address bar don't work as deep links in the new Teams clients.
```

Comment:
Suggestion: Remove the word `Classic` all together for future purposes. Just saying `Teams URLs copied from the browser's address bar are not supported as deeplinks.` should suffice.

Reply:
Updated.